launcher.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
+
+datadir=/app/data
+userdir=~/.elc
+serverfile=servers.lst
+inifile=el.ini
+exename=/app/bin/el.x86.linux.bin
```

Comment:
It doesn't work with other (arm) architectures?

Reply:
Yes it does but I agree the current name is confusing.  I'll modify it in the make file patch for now.